### PR TITLE
Create index.html and sketch.js >> bizarro-devin.setupFiles

### PR DESCRIPTION
**Pull request by JuggernautJf**

### CODE REVIEW

**Comment by JuggernautJf** [comment]
Use Setup Files command

File: src/commands/runAIAgent.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ const { typeRealistically } = require('../util/realisticTyping');
 const { speak } = require('../util/speak');
 const { script } = require('../script');
 
-class SetupLayoutCommand extends Command {
+
+class runAgentCommand extends Command {
```
Comment (dipamsen) [inline src/commands/runAIAgent.js:9]: By convention, class names start with a capital letter, to distinguish them from normal functions. So `RunAgentCommand` is more appropriate here.

File: src/commands/setupFiles.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const Command = require('../lib/command');
+const { createFile } = require('../util/createFile');
+
+class setupFilesCommand extends Command {
```
Comment (dipamsen) [inline src/commands/setupFiles.js:4]: same as above

File: src/util/createFile.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+*/
+async function createFile  (path, content = "",options={}) {
+  const activeFolder = vscode.workspace.workspaceFolders[0];
+  const FilePath = vscode.Uri.joinPath(activeFolder.uri, path);
+  const FileContent = new TextEncoder().encode(content);
+  await vscode.workspace.fs.writeFile(FilePath, FileContent);
+  if (options.show) {
+    const document = await vscode.workspace.openTextDocument(FilePath);
+    await vscode.window.showTextDocument(document);
+  }
```
Comment (dipamsen) [inline src/util/createFile.js:23]: By convention, normal variables in javascript are `camelCase`d, so `FilePath` and `FileContent` can be renamed to `filePath` and `fileContent`.

**Review by dipamsen — COMMENTED** [review]
@JuggernautJf Thank you for contributing! Left some notes/suggestions on variable naming.

**Comment by dipamsen** [comment]
(Note to shiffman : While merging this PR, please make sure to use the `Squash and Merge` option to combine all the commits into a single commit in the main git history.)

**Pull request closed without merging**